src/graphorm/index.ts

```diff
@@ -224,6 +224,9 @@ const obj = new GraphORM({
   Source: {
     requiredColumns: ['id', 'private', 'handle', 'type'],
     fields: {
+      flags: {
+        jsonType: true,
```

Comment:
```suggestion
        jsonType: true,
        transform: (flags: Partial<SourceFlagsPublic>) => {
          return {
            totalPosts: flags?.totalPosts ?? 0,
            totalViews: flags?.totalViews ?? 0,
            totalUpvotes: flags?.totalUpvotes ?? 0,
            featured: flags?.featured ?? false,
          };
        },
```

Reply:
Also if you can add tests, check here for example 🙏 https://github.com/dailydotdev/daily-api/blob/main/__tests__/sources.ts#L2795-L2834

Reply:
I introduced a new function as I realized the properties can grow and the scenario can easily happen in the future. That is to make the process consistent.